.github/workflows/ci.yml

```diff
@@ -20,11 +17,87 @@ on:
   pull_request: {}
 
 env:
+  # Keep this Bazel version in sync with the `versions.check` directive
+  # in our WORKSPACE file.
+  BAZEL_VERSION: '3.7.0'
+  BAZEL_SHA256SUM: 'b7583eec83cc38302997098a40b8c870c37e0ab971a83cb3364c754a178b74ec'
   BUILDTOOLS_VERSION: '3.0.0'
   BUILDIFIER_SHA256SUM: 'e92a6793c7134c5431c58fbc34700664f101e5c9b1c1fcd93b97978e8b7f88db'
   BUILDOZER_SHA256SUM: '3d58a0b6972e4535718cdd6c12778170ea7382de7c75bc3728f5719437ffb84d'
 
 jobs:
+  build:
+    runs-on: ubuntu-16.04
+    needs: lint-python-flake8 # fail fast in case of "undefined variable" errors
+    strategy:
+      fail-fast: false
+      matrix:
+        tf_version_id: ['tf-nightly', 'notf']
+        python_version: ['3.7']
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+          architecture: 'x64'
+      - name: 'Set up Bazel'
+        run: |
+          ci/download_bazel.sh "${BAZEL_VERSION}" "${BAZEL_SHA256SUM}" ~/bazel
+          sudo mv ~/bazel /usr/local/bin/bazel
+          sudo chmod +x /usr/local/bin/bazel
+          cp ./ci/bazelrc_github ~/.bazelrc
+      - name: 'Configure build cache write credentials'
+        env:
+          CREDS: ${{ secrets.BAZEL_CACHE_SERVICE_ACCOUNT_CREDS }}
+          EVENT_TYPE: ${{ github.event_name }}
+        run: |
+          if [ -z "${CREDS}" ]; then
+            printf 'Using read-only cache (no credentials)\n'
+            exit
+          fi
+          if [ "${EVENT_TYPE}" = pull_request ]; then
+            printf 'Using read-only cache (PR build)\n'
+            exit
+          fi
+          printf 'Using writable cache\n'
+          creds_file=/tmp/service_account_creds.json
+          printf '%s\n' "${CREDS}" >"${creds_file}"
+          printf '%s\n' >>~/.bazelrc \
+            "common --google_credentials=${creds_file}" \
+            "common --remote_upload_local_results=true" \
+            ;
+      - name: 'Install Python dependencies'
+        run: |
+          python -m pip install -U pip
```

Comment:
We had a few comments in the Travis version for these lines; worth preserving?
https://github.com/tensorflow/tensorboard/blob/e34809c4e49aedd62aba616a96ab255a251d2816/.travis.yml#L53-L56

Also the `numpy` uninstall was there for a possibly Travis-specific issue.  Do we know if GH Actions has the same problem?  Maybe worth omitting it unless we still need it.

Reply:
Removed (see other comment). Let’s see. From experience, I would expect
GitHub’s defaults to be saner than Travis’s.